ert/data/record/_record.py

```diff
@@ -58,6 +58,8 @@ class RecordType(str, Enum):
     MAPPING_INT_FLOAT = "MAPPING_INT_FLOAT"
     MAPPING_STR_FLOAT = "MAPPING_STR_FLOAT"
     BYTES = "BYTES"
```

Comment:
Separate these constants (empty line?) to signify that they are not to be used for the same?